clang/test/CodeGenSYCL/sycl-intelfpga-field-no-opaque.cpp

```diff
@@ -0,0 +1,74 @@
+// RUN: %clangxx %s -fsycl-device-only -fintelfpga -S -o %t.ll
+// RUN: FileCheck %s --input-file %t.ll
+
+// This test checks that proper IR is generated for kernel field initialization, including
+// 3 cases:
+//   1. initialize pointer field with a global pointer
+//   2. initialize a float field
+//   3. initialize a BitInt field
+// It also checks that the kernel lambda is not inlined for FPGA, i.e. the kernel entry
+// contains `call @NameOfCallOperator(...)`
+
+// Note this is a temporary test for the FPGA-specific use model that will be
```

Comment:
I don't see annotate used anywhere in this test. This test just seems to be testing kernel call/initialization. I would assume there are already tests which test this but I do not know for sure. I don't see the harm of adding this test but I am not certain it is even required. @premanandrao thoughts?

Reply:
I've combined  `sycl-intelfpga-field-no-opaque.cpp` into this test as a sub case, so that all the FE dependency is guarded by one single test.

Reply:
> @premanandrao thoughts?

I really missed that it was not cc1.   That's why I was confused about the __itt check.